.github/ISSUE_TEMPLATE/us-template.md

```diff
@@ -4,28 +4,31 @@ about: User Story
 title: ''
 labels: ''
 assignees: ''
-
```

Comment:
May i ask what is the reason for changing this file?

Reply:
Linting/Formatting